### PR TITLE
Remove "ResetPads()" and do refactoring

### PR DESCRIPTION
**Pull request by flash-fire**
That method was *really*  annoying me.  Tested with the console text input application-- it works since the window isn't visible.

### CODE REVIEW

**Comment by Megamouse** [comment]
What's with those randomly removed spaces in ASCII comments xD
And I think somewhere you add one to the end of the line lol

File: rpcs3/rpcs3qt/instruction_editor_dialog.cpp [hunk 1]
```diff
@@ -84,7 +84,8 @@ instruction_editor_dialog::instruction_editor_dialog(QWidget *parent, u32 _pc, c
 void instruction_editor_dialog::updatePreview()
 {
 	bool ok;
-	ulong opcode = t2_instr->text().toULong(&ok, 16);
```
Comment (Megamouse) [inline rpcs3/rpcs3qt/instruction_editor_dialog.cpp]: this looks like a shitty hack to me so please fix it properly

Comment (flash-fire) [inline rpcs3/rpcs3qt/instruction_editor_dialog.cpp]: I mean it wasn't used so I either was going to delete the line or comment it lol

Comment (Megamouse) [inline rpcs3/rpcs3qt/instruction_editor_dialog.cpp]: but ok is being used
which has now random behaviour if not being used to check for valid number

Comment (flash-fire) [inline rpcs3/rpcs3qt/instruction_editor_dialog.cpp]: so just put that line back in and you're good?

Comment (Megamouse) [inline rpcs3/rpcs3qt/instruction_editor_dialog.cpp]: couldn't you just not assign it to a variable xD ?
that way we don't have a warning

**Pull request merged**